Makefile

```diff
@@ -17,6 +17,9 @@ install:
 test:
 	npm test
 
+bench:
```

Comment:
Would suggest to allow the destination server as an argument (so to allow testing against an arbitrary Graylog/Logstash server).